client/src/components/Join.jsx

```diff
@@ -19,7 +19,7 @@ const Join = ({ setTopLevelState }) => {
 
   return (
     <form>
-      <label forHtml='login'>Add your name</label>
+      <label forHtml='login'>Login As A Guest Add your name</label>
```

Comment:
Should this label read "Login As A Guest Add your name" or just "Login As A Guest"?